API/Services/JwtService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using API.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace API.Services
+{
+    public class JwtService
+    {
+        private readonly string _secret;
+
+        public JwtService(IConfiguration config)
+        {
+            _secret = config.GetSection("Jwt").GetSection("SecretKey").Value;
+        }
+
+        public string GenerateSecurityToken(User user)
+        {
+            var tokenHandler = new JwtSecurityTokenHandler();
+            var key = Encoding.ASCII.GetBytes(_secret);
+            var tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(new[]
+                {
+                    new Claim(ClaimTypes.Email, user.Email),
+                    new Claim(ClaimTypes.Name, user.Username)
+                }),
+                Expires = DateTime.UtcNow.AddDays(30),
```

Comment:
Maybe this could be a constant or environment variable?

Reply:
Updated